tests/integration/docker_images/popular-containers-images.bats

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# Tests for the most popular images from docker hub
+
+setup() {
+	load common
+	cleanDockerPs
+	runtimeDocker
+}
+
+@test "run hello world in java in an openjdk container" {
+	$DOCKER_EXE run --rm -i openjdk bash -c "echo 'public class HW{public static void main(String[]a){System.out.println(\"HelloWorld\");}}' > HW.java && javac HW.java && java HW"
+}
+
+@test "check kong configuration file is valid" {
+	$DOCKER_EXE run --rm -i kong bash -c "cp /etc/kong/kong.conf.default /usr/local/kong/kong.conf; kong check /usr/local/kong/kong.conf | grep "valid""
+}
+
+@test "create a directory in an aerospike container" {
+	$DOCKER_EXE run --rm -i aerospike bash -c "mkdir /home/test | ls /home | grep "test""
+}
+
+@test "hello world in an alpine container" {
+	$DOCKER_EXE run --rm -i alpine sh -c "echo 'Hello, World'"
+}
+
+@test "run a celery container" {
+	$DOCKER_EXE run --rm -i celery bash -c "python -m timeit \"[i for i in range(1000)]"\"
+}
+
+@test "search job_spec in the  help option of a bonita container" {
+	$DOCKER_EXE run --rm -i -e "TENANT_LOGIN=testuser" -e "TENANT_PASSWORD=mysecretword" bonita bash -c "help | grep "job""
+}
+
+@test "start redis server with a certain port" {
+	$DOCKER_EXE run --rm -i redis bash -c "timeout 10 redis-server --port 7777 | grep "7777""
+}
+
+@test "check os-core-update bundle in a clearlinux container" {
+	$DOCKER_EXE run --rm -i clearlinux bash -c "ls /usr/share/clear/bundles | grep "os-core-update""
+}
+
+@test "run golang container" {
+	$DOCKER_EXE run --rm -i golang bash -c "echo -e 'package main\nimport \"fmt\"\nfunc main() { fmt.Println(\"hello world\")}' > hello-world.go && go run hello-world.go && go build hello-world.go"
+}
+
+@test "create files in an oraclelinux container" {
+	$DOCKER_EXE run --rm -i oraclelinux bash -c 'for NUM in `seq 1 1 10`; do touch $NUM-file.txt; done; ls -l /1-file.txt'
```

Comment:
You could make that just `seq 10`. But I'm not clear why you are creating 10 files then checking the first only? Could this test be simple:

- create file
- check it exists?